src/main/java/com/tictactoe/HumanPlayer.java

```diff
@@ -17,6 +16,12 @@ public String getInput() {
         return this.input;
     }
 
+    public void addSymbol(String symbol) {
+        this.symbol = symbol;
+    }
+
+    public void addBoard(Board board) {}
```

Comment:
Why does `HumanPlayer` have this method? In general, it seems a bit weird that a player is given a board and a game is given a board as well.